src/app/policies/views/PolicyDetailView.vue

```diff
@@ -63,12 +64,53 @@
           <KCard>
             <h2>{{ t('policies.detail.affected_dpps') }}</h2>
 
-            <PolicyConnections
-              class="mt-4"
-              :mesh="route.params.mesh"
-              :policy-name="route.params.policy"
-              :policy-path="route.params.policyPath"
-            />
+            <div class="mt-4">
+              <KInput
+                id="dataplane-search"
+                :model-value="route.params.dataplane"
+                type="text"
+                :placeholder="t('policies.detail.dataplane_input_placeholder')"
+                required
+                data-testid="dataplane-search-input"
+                @input="route.update({ dataplane: $event })"
```

Comment:
```suggestion
                @input="(str: string) => route.update({ dataplane: str })"
```

Avoiding the weird `$event` where there's no benefit to use it? Bit clearer that the emitted thing is not an event but a string also.

Reply:
It doesn’t really matter which style we use. I’m gonna leave this as-is.